pkg/scheduler/api/resource_info.go

```diff
@@ -193,13 +194,24 @@ func (r *Resource) Get(rn v1.ResourceName) float64 {
 	}
 }
 
+// Skip checking "pods" resource.
+// All pods request one "pods" resource now, no need to check it
+var emptyCheckIgnoredScalarResources = sets.NewString(string(v1.ResourcePods))
+
+func IsEmptyCheckIgnoredScalarResource(name v1.ResourceName) bool {
```

Comment:
It's better to change a readable function name.  For example, `IsIgnoredScalarResource` or `IsInBlacklist`

Reply:
Done